src/Components/Login/Login.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import './Login.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props) 
+    this.state = {
+      name: '',
+      quote: '',
+      skillLevel: ''
+    }
+  }
+  
+  handleChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value })
+  }
+
+  submitLoginHandler = () => {
```

Comment:
Once we start working on the error handling for the inputs those methods can be placed within the submitLoginHandler, and we will probably want to put the two methods taken from props into the conditional so that they only run if the user has filled out the inputs correctly. Also we could do some deconstructing here for those props - like on 13 have `const { handleLogin, handleUserInfo } = this props` and then just use those new variables to call the methods